docs/README.md

```diff
@@ -1,30 +1,35 @@
-# Model Server for PyTorch Documentation
+# TorchServe
+
+TorchServe is a flexible and easy to use tool for serving PyTorch models.
 
 ## Basic Features
 
-* [Serving Quick Start](../README.md#serve-a-model) - Basic server usage tutorial
-* [Model Archive Quick Start](../model-archiver#creating-a-model-archive) - Tutorial that shows you how to package a model archive file.
-* [Installation](../README.md##install-torchserve) - Installation procedures
-* [Serving Models](server.md) - Explains how to use `torchserve`.
-  * [REST API](rest_api.md) - Specification on the API endpoint for TorchServe
-  * [gRPC API](grpc_api.md) - Specification on the gRPC API endpoint for TorchServe
-* [Packaging Model Archive](../model-archiver/README.md) - Explains how to package model archive file, use `model-archiver`.
+* [Serving Quick Start](https://github.com/pytorch/serve/blob/master/README.md#serve-a-model) - Basic server usage tutorial
```

Comment:
NIT: I think the relative links were done for generated docs. Not quite sure what the workflow was, but worth checking before we merge this change

Reply:
So it's relative links to anything inside the docs/ folder if anything out then absolute links are the convention - confirmed with the Pytorch docs team on this